### PR TITLE
use vale and Red Hat style to validate documentation

### PR DESCRIPTION
**Pull request by gebhardtr**
WIP:
- [ ] ignore style for docs that come from legal (terms) (TBD)?
- [ ] (temporary) add Lightspeed specific vocab but ultimately add it to the [Red Hat style project](https://github.com/redhat-documentation/vale-at-red-hat)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
-## Building docs
+## Get started
+
+### Install dependencies
 
 ```bash
 python3 -m venv venv
 source venv/bin/activate
-python3 -m pip install -r docs/requirements.txt
+python3 -m pip install -r docs/requirements-dev.txt
```
Comment (scottharwell) [inline README.md:10]: Isn't the `requirements-dev.txt` file in the root?

**Comment by scottharwell** [comment]
I think that once we merge this, we should expand the scope with markdown lint, too.  I already have some rules applied through the docs.

File: .vale.ini [hunk 2]
```diff
@@ -0,0 +1,7 @@
+StylesPath = .vale/styles
+MinAlertLevel = suggestion
+Packages = RedHat
```
[the remark this answers is not in the document]
Comment (gebhardtr) [reply .vale.ini:3]: Yep! It should be after `vale sync`

File: README.md (hunk 1 above)
Comment (gebhardtr) [reply README.md:10]: nice catch, thank you

File: .vale.ini (hunk 2 above)
[the remark this answers is not in the document]
Comment (gebhardtr) [reply .vale.ini:3]: did you run it via:
```bash
tox -e linters
```

**Pull request closed without merging**